target/linux/ar71xx/image/generic.mk

```diff
@@ -6,6 +6,19 @@ define Device/ap90q
 endef
 TARGET_DEVICES += ap90q
 
+define Device/AP531B0
+  DEVICE_TITLE := Rockeetech AP531B0 board
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2
+  BOARDNAME = AP531B0
+  ROOTFS_SIZE := 14528k
```

Comment:
You don't need to define kernel and rootfs sizes here as your board uses kernel-rootfs mtd partition order.
Please, take the `cf-e520n` as example and use same approach for image definition.

You have also changed file attributes (`100644 → 100755`) which is **wrong** - please, fix it.